CHANGELOG.md

```diff
@@ -1,20 +1,29 @@
 # Change Log
 
-All notable changes to the mirrord's cli, agent, protocol, extensions will be documented in this file.
-Previous versions had CHANGELOG per component, we decided to combine all repositories to a mono-repo with one CHANGELOG.
+All notable changes to this project will be documented in this file.
 
-Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in <https://github.com/metalbear-co/mirrord/tree/main/changelog.d/>.
 
-### Fixed
+<!-- towncrier release notes start -->
+
+## [3.32.0](https://github.com/metalbear-co/mirrord/tree/3.32.0) - 2023-03-07
```

Comment:
Is this on purpose? Shouldn't this only happen on release? Can we maybe have it automatically happen in release.yaml?